lib/tasks/scripts/split_bathinda_and_mansa.rb

```diff
@@ -0,0 +1,61 @@
+class SplitBathindaAndMansa
+  def self.call(*args)
+    return "Cannot run this outside India" unless CountryConfig.current[:name] == "India"
+
+    new.call
+  end
+
+  def initialize
+    @bathinda_and_mansa = FacilityGroup.find_by!(name: "Bathinda and Mansa")
+
+    # These facility groups need to be created before running this rake task
```

Comment:
Is this a manual step the runner of this task will do? Or will it be done in the UI? Or should we just do it in the task here itself?

Reply:
This is easier to do it from the UI and not worry about all the region stuff.